cdap-docs/developers-manual/source/building-blocks/datasets/fileset.rst

```diff
@@ -317,6 +317,54 @@ For example, give these arguments when starting the MapReduce through a RESTful
     "dataset.totals.output.partition.key.league" : "nfl"
   }
 
+Incrementally Processing Partitions of a PartitionedFileSets in MapReduce
+=========================================================================
+
+Another way to process a partitioned file set from a MapReduce is via the beta feature of incrementally
+consuming its partitions. Using this, a MapReduce job can run and process the files of all partitions
+committed to a particular partitioned file set since the last run of the MapReduce job. Currently, the
```

Comment:
I think this is phrased with a little bit of assumptions. Perhaps rephrase: 

One way to process a partitioned file set is with a MapReduce program that runs repeatedly and, in every run, reads the partitions that have been added since its previous run. This requires that the MapReduce program persists between runs what partitions have already been consumed.  An easy way to do that is the ``BatchPartitionConsumer``, an experimental feature introduced in CDAP 3.1.0. The MapReduce program is responsible for persisting the state of this consumer. In the following example, the state is persisted to a row in a KeyValue Table, but other types of Datasets can also be used::